### ISSUE TITLE
Produced DNGs have a missing or invalid BitsPerSample according to dng_validate

### ISSUE THREAD

**Issue by RazrFalcon** [dnglab/dnglab#455]
Adobe's DNG SDK has a tool called `dng_validate` with a self explanatory name and it prints the following output for DNGs created by `dnglab`:

```
*** Warning: IFD 0 NewSubFileType has unexpected type (Short) ***
*** Warning: SubIFD 1 NewSubFileType has unexpected type (Short) ***
*** Warning: SubIFD 2 NewSubFileType has unexpected type (Short) ***
*** Error: Missing or invalid BitsPerSample (IFD 0) ***
*** Error: Missing or invalid BitsPerSample (SubIFD 2) ***
```

Also, Apple Preview cannot open dnglab-produced DNGs (compressed and uncompressed), but can open Adobe DNG Convert DNGs just fine. Maybe because of invalid `BitsPerSample`?

**Comment by cytrinox** [comment]
Thanks, this is fixed in [46edf19](https://github.com/dnglab/dnglab/commit/46edf1954f525845f42a3f360b9d31d54db23b82)

**Comment by RazrFalcon** [comment]
Can confirm that it fixed Apple Preview support. Thanks!
